.github/workflows/AssignCommitter.yml

```diff
@@ -0,0 +1,47 @@
+name: Assign Committer
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  assign-reviewer:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.draft != true }}
+    steps:
+      - name: Add Assignee
+        uses: actions/github-script@v7
+        with:
+          script: |
+            if (context.payload.pull_request.assignees.length === 0) {
+              // Find suitable assignees
```

Comment:
For this section ("Find suitable assignees"), can you implement the following?

1. Use the GitHub API to retrieve the contents of this file: https://github.com/JuliaLang/pr-assignment/blob/main/users.txt
2. Once you have the file contents, I'd recommend processing it like this.
    1. Split into lines.
    2. Strip trailing whitespace.
    3. Ignore all lines that start with `#`
    4. Strip comments, e.g. if a line is `@myname # some comment`, discard everything after `#`
    5. Require that usernames have the `@` at the beginning, for consistency with how GitHub `CODEOWNERS` files behave.

Reply:
Can you also add a comment in this YAML file that says something along the lines of this:

> Note to JuliaLang committers: If you would like to add yourself to the pool of committers that are available to be assigned to triage new PRs, please make a PR to this repo: https://github.com/JuliaLang/pr-assignment

Reply:
Something like this regex should work for parsing the username lines in `users.txt`: `^@([A-Za-z0-9\-]+?)$`

Taken from https://github.com/JuliaLang/pr-assignment/blob/b9a8017d33574462b9a502b279907d9b8e003554/.github/workflows/lint.yml#L51-L51

Reply:
Done